### PR TITLE
Add WND preprocess readme

### PR DESCRIPTION
**Pull request by jenniew**
Update preprocess source file and add readme.
Related issue: https://github.com/intel-analytics/analytics-zoo/issues/4160

### CODE REVIEW

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Prepare the environment
+We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster (yarn-client mode only).
+```
+conda create -n zoo python=3.7  # "zoo" is the conda environment name, you can use any name you like.
+conda activate zoo
+pip install --pre --upgrade analytics-zoo
+```
+
+__Note:__ As we test, Spark 3.0.0 will have performance benefit over the default Spark 2.4.3 and you can download the Analytics Zoo package built with Spark 3.0.0 from [here](https://sourceforge.net/projects/analytics-zoo/files/zoo-py/).
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/wnd/README.md:13]: remove this line? As we haven't fully tested the performance for WND

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+```bash
+python wnd_preprocessing.py \
+    --executor_cores 36 \
+    --executor_memory 50g \
+    --days 0-1 \
+    --input_folder /path/to/the/folder/of/parquet_files \
+    --output_foler /path/to/save/preprocessed/folder/of/parquet_files \
+    --frequency_limit 15 \
+    --cross_sizes 10000, 10000
+
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/wnd/README.md:35]: remove this empty line

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+* `executor_cores`: The number of cores to use on each node. Default to be 48.
+* `executor_memory`: The amount of memory to allocate on each node. Default to be 240g.
+* `num_nodes`: The number of nodes to use in the cluster. Default to be 8.
+* `driver_cores`: The number of cores to use for the driver. Default to be 4.
+* `driver_memory`: The amount of memory to allocate for the driver. Default to be 36g.
+* `days`: The day range for data preprocessing, such as 0-23 for the full Criteo dataset, 0-1 for the first two days, etc. Default to be 0-23.
+* `frequency_limit`: Categories with frequency below this value will be omitted from encoding. We recommend using 15 when you preprocess the full 1TB dataset. Default to be 15.
+* `input_folder`: The path to the folder of parquet files, either a local path or an HDFS path.
+* `output_folder`: The path to save the preprocessed data to parquet files. HDFS path is recommended.
+* `cross_sizes`: bucket sizes for cross columns (`c14-c15` and `c16-c17`)
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/wnd/README.md:79]: The bucker sizes for cross columns (`c14-c15` and `c16-c17`) separated by comma. Default to be 10000, 10000.

File: pyzoo/zoo/examples/friesian/feature/wnd/wnd_preprocessing.py [hunk 4]
```diff
@@ -117,29 +127,26 @@ def _parse_args():
...
+
+        test_data = FeatureTable.read_parquet(
+            os.path.join(args.input_folder, "day_23_test.parquet"))
+        preprocess_and_save(test_data, idx_list, os.path.join(args.output_folder, "test_parquet"))
+    else: # Sample data
+        data = FeatureTable.read_parquet(paths)
+        preprocess_and_save(data, idx_list, os.path.join(args.output_folder, "data_parquet"))
+
+    time_end = time()
+    print("Train data loading and preprocessing time: ", time_end - time_start)
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/wnd/wnd_preprocessing.py:151]: Change to `Total preprocessing time` as now we have train and test

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+```bash
+python wnd_preprocessing.py \
+    --cluster_mode standalone \
+    --master spark://master-url:port \
+    --executor_cores 56 \
+    --executor_memory 240g \
+    --num_executor 8 \
+    --days 0-23 \
+    --input_folder /path/to/the/folder/of/parquet_files \
+    --output_foler /path/to/save/preprocessed/folder/of/parquet_files \
```
Comment (hkvision) [inline pyzoo/zoo/examples/friesian/feature/wnd/README.md:48]: /path/to/the/folder/to/save/preprocessed/parquet_files ?

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md (hunk 1 above)
Comment (jenniew) [reply pyzoo/zoo/examples/friesian/feature/wnd/README.md]: updated.

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md (hunk 2 above)
Comment (jenniew) [reply pyzoo/zoo/examples/friesian/feature/wnd/README.md]: updated.

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md (hunk 3 above)
Comment (jenniew) [reply pyzoo/zoo/examples/friesian/feature/wnd/README.md]: updated.

File: pyzoo/zoo/examples/friesian/feature/wnd/wnd_preprocessing.py (hunk 4 above)
Comment (jenniew) [reply pyzoo/zoo/examples/friesian/feature/wnd/wnd_preprocessing.py]: updated

File: pyzoo/zoo/examples/friesian/feature/wnd/README.md (hunk 5 above)
Comment (jenniew) [reply pyzoo/zoo/examples/friesian/feature/wnd/README.md]: updated.

**Review by hkvision — APPROVED** [review]

**Comment by jenniew** [comment]
Jenkins: http://10.239.176.111:18888/job/ZOO-PR-Validation/5957/

**Pull request merged**